src/modules/dashboard/components/CoinMachine/CoinMachine.tsx

```diff
@@ -82,6 +83,12 @@ const CoinMachine = ({
   const { transactionHash } = useParams<{
     transactionHash: string;
   }>();
+  const {
+    data: salePeriodsData,
+    loading: salePeriodsLoading,
+  } = useSubgraphCoinMachinePeriodsQuery({
+    variables: { colonyAddress: colonyAddress.toLowerCase() },
+  });
```

Comment:
I couldn't get this to display anything. I make two periods go by:
![Screenshot from 2021-08-30 14-31-08](https://user-images.githubusercontent.com/1193222/131333325-2e787817-9143-492a-8ce1-3e54875bb3d7.png)

Please notice the buy price, that's because I've bought all tokens in the previous periods.q

However, looking at query directly, it does seem that it's returning values:

![Screenshot from 2021-08-30 14-33-31](https://user-images.githubusercontent.com/1193222/131333409-7116505e-95cd-4162-8b45-e71fbc03ce28.png)

Please investigate what's going on

Reply:
I've re-tested this and now seems to be working, no idea what the issue was the first time around.

![Screenshot from 2021-08-30 17-44-56](https://user-images.githubusercontent.com/1193222/131357764-63b38bb8-0a47-4493-a6e4-d8119774730b.png)

You can consider this fixed